libs-scala/crypto/src/jmh/scala/com/daml/MacPrototypeBenchmark.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.crypto
+import org.openjdk.jmh.annotations._
+
+import java.nio.charset.Charset
+import javax.crypto.spec.SecretKeySpec
+import scala.util.Random
+
+@State(Scope.Benchmark)
+@Warmup(iterations = 3, time = 1)
+@Measurement(iterations = 5, time = 5)
+class MacPrototypeBenchmark {
+
+  var key = ""
+  var value = ""
+  val charset = Charset.forName("UTF-8")
```

Comment:
I guess that's the same as `StandardCharsets.UTF_8`

Reply:
oh thanks, i was looking for that ;)